website/docs/docs/user-interface/dashboard.md

```diff
@@ -96,3 +100,19 @@ Clicking on a tag pill for a provider version tag will make the following menu a
 
 - Copy tag URL - selecting this menu item will copy the URL of the application version tag into the clipboard.
 - Copy tag name - selecting this menu item will copy the name of the tag into the clipboard.
+
+## Matrix 
+
+"Matrix" displays pacts and verifications of the selected integration.
```

Comment:
How about:

The "Matrix" displays a table that shows the verification status of each combination of consumer and provider version from the selected integration.

Reply:
It's way accurate, thanks!